lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt

```diff
@@ -157,6 +162,25 @@ fun PreferencesOverflowMenu() {
     }
 }
 
+@Composable
+fun PreferencesDebugWarning() {
+    val enableDebug by preferenceManager().enableDebugMenu.observeAsState()
+    if (BuildConfig.DEBUG && !enableDebug) {
```

Comment:
Please use a guard clause instead.

```suggestion
    val enableDebug by preferenceManager().enableDebugMenu.observeAsState()
    if (enableDebug) return
    // ui code here
```